source/vibe/http/common.d

```diff
@@ -469,20 +488,25 @@ final class ChunkedOutputStream : OutputStream {
 	{
 		assert(!m_finalized);
 		if( m_buffer.data.length > 0 ) flush();
+		ubyte[] buffer = null;
+		auto getBytes = (size_t size) {
+			if (buffer is null || buffer.length < size)
+				buffer = new ubyte[](size);
+				return buffer[0..size];
+			};
```

Comment:
Actually, I have been looking at the wrong thing all along - guess I didn't read the type toroughly enough and just assumed a `std.array.Appender`.  Yeah, if we could have a new method for this use case, that would be a nice solution.